src/azure-cli/azure/cli/command_modules/serviceconnector/custom.py

```diff
@@ -300,7 +300,9 @@ def connection_create(cmd, client,  # pylint: disable=too-many-locals,too-many-s
                       config_store=None,                                     # Resource.AppConfig
                       namespace=None,                                        # Resource.EventHub
                       webpubsub=None,                                        # Resource.WebPubSub
-                      signalr=None):                                         # Resource.SignalR
+                      signalr=None,                                          # Resource.SignalR
+                      customized_keys=None,
```

Comment:
move it up to the feature related parameters, on top of resource related parameters

similar below